docs/graphs/graph-queries/shortest-path-queries/shortest-path-examples.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 30
+title: Shortest Path Examples
+---
+
+These examples are based on the [Traversal Graph](../../graph-examples/example-graphs.md#the-traversal-graph). Create a graph based on the example graph to follow along.
+
+![traversal graph](/img/graphs/traversal_graph.png)
+
+## Shortest Path Example
+
+You can query the shortest path from **A** to **D** using either the named graph syntax or the collection sets syntax. The results are identical. For information about running queries, refer to [Running Queries](../../../queries/running-queries).
+
+### Named Graph
+
+```sql
+FOR v, e IN OUTBOUND SHORTEST_PATH 'circles/A' TO 'circles/D' GRAPH 'traversalGraph' 
+  RETURN [v._key, e._key]
+```
+
+This query uses the named graph `'traversalGraph'` and the `OUTBOUND` keyword to traverse from the start vertex `'circles/A'` to the target vertex `'circles/D'`. The `SHORTEST_PATH` keyword ensures that only the shortest path is returned.
+
+The `RETURN` statement returns an array containing the `_key` attributes of both the vertex (`v._key`) and the edge (`e._key`) of each step along the path. Because this query is using a named graph, the edge collections are inferred from the graph and do not need to be specified explicitly.
+
+### Collection Sets
+
+```sql
+FOR v, e IN OUTBOUND SHORTEST_PATH 'circles/A' TO 'circles/D' edges 
+  RETURN [v._key, e._key]
+```
+
+This query specifies the edge collection to traverse (`edges`). The `OUTBOUND` keyword is used to traverse in the outbound direction from the start vertex `'circles/A'` to the target vertex `'circles/D'`. The `SHORTEST_PATH` keyword ensures that only the shortest path is returned.
+
+The `RETURN` statement returns an array containing the `_key` attributes of both the vertex (`v._key`) and the edge (`e._key`) of each step along the path.
+
+## Traverse Mixed Directions Example
+
+You can also compute shortest paths based on documents found in collections. You can use either the named graph syntax or the collection sets syntax, the results are identical. For information about running queries, refer to [Running Queries](../../../queries/running-queries).
```

Comment:
@diana-macrometa here we not specified an example plus the explanation  is also of the generic Shortest path example. 
We will need to add an example for this but the example will need multiple edge collection so the "traversalGraph" example which has been used in this whole page won't work for mixed directions example

Reply:
Maybe the Cities Graph would work?

Reply:
The problem with city Graph is that although there are three edgecollections but none of them have any common edges.. so adding any mixed direction traversal won't make any difference there. We will need to create some sample data but then the length will become big for this example.. anyways I will get you some working example for this tomorrow.. and then you can decide where to keep it and all

Reply:
Check the example in Dino's SDK PR: https://github.com/Macrometacorp/docs/pull/779

Reply:
Nope that won't work it uses a single edge collection

Reply:
I will create a simple sample data for this today

Reply:
@diana-macrometa can we remove this section:
`## Traverse Mixed Directions Example

You can also compute shortest paths based on documents found in collections. You can use either the named graph syntax or the collection sets syntax, the results are identical. For information about running queries, refer to [Running Queries](../../../queries/running-queries).`

from shortest path examples for now? and later, we can add it you can assign a doc ticket to me saying to add a sample dataset and query because I was not able to find time for this today

Reply:
I would prefer not to remove the section, because then it won't match the other query topics which have the Traverse Mixed Directions sections. Is it good enough for now? I mean, this content has been this way since forever. 

As long as everything is updated by the end of the week, it's all good.

Reply:
got it